### DISCUSSION TITLE
Test suite expected status?

### DISCUSSION THREAD

**Discussion by clbr** [cc65/cc65#1873] [Q&A]
I ran "make test", and got 115 failures. I do see some issues open where tests are expected to fail currently, but wondering what's the expected number?

**Comment by mrdudz** [comment]
That is the output of a single test program (some printf thing iirc) - it does not refer to the testbench as such.

If some test gives a different result than expected, "make test" will fail, ie "make" will report an error and the testbench will stop. If you are not sure, run "make test" again when its done - in case of a problem it will fail again.

**Comment by clbr** [comment]
Oh, thanks. It's indeed the todo/sprintf-test.c output, and not a summary like it looked like. Make returned 0 as its exit status. Could that test's output be changed so it doesn't confuse users? E.g. prefixing that line with "sprintf-tests: ".

The test suite took 5min 30s, when a few years back I recall it used to take 1min. That's good, lots of tests added ;)

**Comment by mrdudz** [comment]
> Could that test's output be changed so it doesn't confuse users? E.g. prefixing that line with "sprintf-tests: ".

sure, feel free to make a PR :)
